README.md

```diff
@@ -69,7 +71,61 @@ func main() {
 }
 ```
 
-More documentation will be coming as the API stabilises! Things are still in flux.
```

Comment:
I think it's useful to warn people that the library interface is still unstable, although maybe not just as an excuse for why there's not much documentation yet.